main.py

```
@@ -52,6 +53,64 @@
 # Background service manager
 background_manager = BackgroundServiceManager()
 
+PRIMARY_WORKER_LOCK_PATH = "/tmp/crypto_primary_worker.lock"
```

Ruff (0.14.2)

Probable insecure usage of temporary file or directory: "/tmp/crypto_primary_worker.lock"

(S108)